arangod/RocksDBEngine/RocksDBHotBackup.cpp

```diff
@@ -156,7 +163,37 @@ std::string RocksDBHotBackup::loadAgencyJson(std::string filename) {
 #else
   return basics::FileUtils::slurp(filename);
 #endif
+}
+
+ResultT<BackupMeta> RocksDBHotBackup::getMeta(std::string const& id) {
+  try {
+    std::string directory = rebuildPath(id);
+
+    std::string metaString =
+      basics::FileUtils::slurp(
+        directory + TRI_DIR_SEPARATOR_CHAR + "META");
+
+    std::shared_ptr<VPackBuilder> metaBuilder =
+      arangodb::velocypack::Parser::fromJson(metaString);
+
+    return BackupMeta::fromSlice(metaBuilder->slice());
 
+  } catch (std::exception const& e) {
+    return ResultT<BackupMeta>::error(TRI_ERROR_HOT_BACKUP_INTERNAL, e.what());
+  }
+}
+
+Result RocksDBHotBackup::writeMeta(std::string const& id, BackupMeta const& meta) try {
+  std::string directory = rebuildPath(id);
+  std::string versionFileName = directory + TRI_DIR_SEPARATOR_CHAR + "META";
+  VPackBuilder metaBuilder;
+  meta.toVelocyPack(metaBuilder);
+  basics::FileUtils::spit(versionFileName, metaBuilder.toJson(), true);
+  return Result{};
+} catch(std::exception const& e) {
+  _errorMessage =
+    std::string("RocksDBHotBackupCreate caught exception: ") + e.what();
```

Comment:
This function is RocksDBHotBackup::writeMeta() ... is that what the _errorMessage should contain?